SlipeServer.Packets/Definitions/Ped/PedSyncPacket.cs

```diff
@@ -0,0 +1,158 @@
+            public float Armor { get; set; }
+            public bool IsOnFire { get; set; }
+            public bool IsInWater { get; set; }
+        }
+
+        private List<SyncData> Syncs;
+
+        public PedSyncPacket()
+        {
+            this.Syncs = new List<SyncData>();
+        }
+
+        public override byte[] Write()
+        {
+            var builder = new PacketBuilder();
+
+            foreach (var data in this.Syncs)
+            {
+                if (data.Send)
+                {
+                    // Vehicle ID
+                    builder.Write(data.Model);
+
+                    // Sync time context
+                    builder.Write(data.TimeSyncContext);
+
+                    // Flags
+                    builder.Write(data.Flags);
+
+                    // Position
+                    if ((data.Flags & 0x01) != 0)
+                    {
+                        builder.Write(data.Position.X);
+                        builder.Write(data.Position.Y);
+                        builder.Write(data.Position.Z);
+                    }
+
+                    // Rotation
+                    if ((data.Flags & 0x02) != 0)
+                        builder.Write(data.Rotation);
+                    
+
+                    // Velocity
+                    if ((data.Flags & 0x04) != 0)
+                    {
+                        builder.Write(data.Velocity.X);
+                        builder.Write(data.Velocity.Y);
+                        builder.Write(data.Velocity.Z);
+                    }
+
+                    // Health, armor, on fire, is in water
+                    if ((data.Flags & 0x08) != 0)
+                        builder.Write(data.Health);
+                    if ((data.Flags & 0x10) != 0)
+                        builder.Write(data.Armor);
+                    if ((data.Flags & 0x20) != 0)
+                        builder.Write(data.IsOnFire);
+                    if ((data.Flags & 0x40) != 0)
+                        builder.Write(data.IsInWater);
+                }
+            }
+
+            return builder.Build();
+        }
+
+        public override void Read(byte[] bytes)
+        {
+            var reader = new PacketReader(bytes);
+            while ((reader.Size / 8) > 32)
+            {
+                SyncData data = new SyncData();
+                data.Send = false;
+
+                data.Model = reader.GetByte();
+
+                data.TimeSyncContext = reader.GetByte();
+
+                byte flags = 0;
+
+                flags = reader.GetByte();
+
+                data.Flags = flags;
+
+                // Position
+                if ((flags & 0x01) != 0)
+                {
+                    Vector3 position = reader.GetVector3WithZAsFloat();
+                    data.Position = position;
+                }
+
+                // Rotation
+                if ((flags & 0x02) != 0)
+                {
+                    data.Rotation = reader.GetByte();
```

Comment:
Rotation can't be 1 byte, must be a float?

```suggestion
                    data.Rotation = reader.GetFloat();
```